src/routes/(inner)/docs/contributions/documentation/+page.svelte

```diff
@@ -0,0 +1,72 @@
+		<p>The simplest way to start documenting your component is to add JSDoc style comment blocks immediately prior to your exported props.  
+			Where the intializer value of your prop is computed or derived from a context, an additional <code>@type</code> tag is needed to help Sveld determine the type.
+		</p>
+		<p>Sveld doesn't support generics at this point, so please note the alternate way we have specified the type within the <code>@type</code> tag.</p>
+		<CodeBlock language="js" code={`// Props (Listbox)
+/** Provide a writable store to maintain list selection.
+ * @type {writeable(any) | writable(any[])}
+ */
+export let selected: Writable<any> = writable(undefined);
+/** Provide classes to set vertical item spacing. */
+export let space: string = 'space-y-1';`}/>
+	<p>Slot documentation is provided by placing a JSDoc comment block at the top (by convention) of your script block.  The default slot is not able to have a description applied at this point, 
+		but if it would be helpful to end developers to describe it, consider adding it to the description of the component. 
+	</p>
+	<p>The hyphen between the name of the named slot and the description as well as the preceding <code>//Slots</code> comment block, are both critical for the Sveld parser to successfully extract the information.</p>
+	<CodeBlock language="js" code={`//Slots
+/**
+ * @slot lead - Provide a leading element, such as an icon.
+ * @slot content - Provide the alert message text.
+ */`}/>
+	</section>
+	<p>Event documentation is created by adding a JSDoc block immediately prior to the dispatch of your event. Sveld will automatically detect forwarded events.</p>
+	<CodeBlock language="js" code={`/** @event {{ event: DragEvent }} dragover - When a file is dragged over the component. */
+dispatch('dragover', event);`}
+	<hr />
+	<section class="space-y-4">
+		<h2>Creating the doc page</h2>
+		<p>Now that we have added all the information to the component, it's time to create the doc page that displays all of the information derived by Sveld.</p>
+		<p>Grab the boilerplate template for a new documentation page from here <code>/src/routes/(inner)/template/+page.svelte</code> and copy it to the appropriate place with lowercase names and dashes for the enclosing folder
+		e.g. <code>/routes/components/your-new-component/+page.svelte</code>.</p>
+		<p>The key to getting all the Sveld information is to adjust the import of your component at the top of the file, ensuring that you preserve the <code>?raw&sveld</code> url params. Now we can pass it into the 
+			<code>DocsShellSettings.components</code> array.</p>
+		<p>All the automated docs should be coming through in the tabs for your component, now is a good time to review that they are all coming through and appearing as expected.</p>
+		<p>Unfortunately not everything can be automated, so for the rest of the information, you need to fill out the props on the <code>DocsShellSettings</code> be replacing the template defaults.  
```

Comment:
A warning style might be good, maybe even also a h3 "manual documentation checklist" or something

Reply:
I'll wait till things have settled a little more before adding to this